src/app/swap/get-active-loopd.ts

```diff
@@ -0,0 +1,36 @@
+import { getActiveLnd } from "@services/lnd/utils"
+import { SwapErrorNoActiveLoopdNode } from "@domain/swap/errors"
+import { BTC_NETWORK, getSwapConfig, getLoopConfig } from "@config"
+import { LoopdInstanceName } from "@domain/swap"
+
+export const getActiveLoopd = (): LoopdConfig => {
+  const activeOffChainNode = getActiveLnd()
+  if (activeOffChainNode instanceof Error) throw SwapErrorNoActiveLoopdNode
+  switch (activeOffChainNode.name) {
+    case "LND1": {
+      return LND1_LOOP_CONFIG
+    }
+    case "LND2": {
+      return LND2_LOOP_CONFIG
+    }
+    default: {
+      throw SwapErrorNoActiveLoopdNode
+    }
+  }
+}
+
+export const LND1_LOOP_CONFIG: LoopdConfig = {
```

Comment:
does this need to be exported?
```suggestion
const LND1_LOOP_CONFIG: LoopdConfig = {
```

Reply:
i use it in the swap integration test to specifically target the lnd1-loop node